### PR TITLE
cdc-tipocket: support to enable kafka sink

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### What problem does this PR solve? 
Support to enable kafka sink for cdc.

### What is changed and how does it work?
Add an option for enabling kafka sink.

### Check List 

Tests 

 - E2E test

Code changes

 - Has Go code change

### Does this PR introduce a user-facing change?:

No.

### CODE REVIEW

File: pkg/test-infra/cdc/recommend.go [hunk 1]
```diff
@@ -163,6 +170,147 @@ func newCDC(ns, name string) *CDC {
...
+			Spec: batchv1.JobSpec{
+				Template: corev1.PodTemplateSpec{
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name:            "cdc-kafka-consumer",
+								Image:           buildCDCImage("cdc-kafka-consumer"),
+								ImagePullPolicy: corev1.PullAlways,
+								Command: []string{
+									"/cdc_kafka_consumer",
```
Comment (mahjonp) [inline pkg/test-infra/cdc/recommend.go:300]: What's the cdc_kafka_consumer  function ?

**Comment by mahjonp** [comment]
https://github.com/pingcap/tipocket/blob/13d9da606eebbe76f3a38122e24d04555d5ef753/pkg/test-infra/clusters.go#L146-L155

We use the composition way to build a cluster, so downstream tidb cluster will also be deployed if you set EnableKafka now. You should also change here.

File: pkg/test-infra/cdc/recommend.go (hunk 1 above)
Comment (5kbpers) [reply pkg/test-infra/cdc/recommend.go:380]: Pull data from kafka, decode and sink them to downstream tidb.

**Comment by 5kbpers** [comment]
@mahjonp Actually we need downstream tidb.

**Pull request merged**